src/modules/missionHelper/i18n/pl_PL.json

```diff
@@ -0,0 +1,75 @@
+{
+	"diyMission": "This mission seems to be a large scale alliance mission!",
+	"expansions": "Spread | Spreads",
+	"followups": "Follow-up Mission | Follow-up Missions",
+	"generated_by": "Generated by",
+	"noVehicleRequirements": [
+		"water_needed",
+		"patient_critical_care"
+	],
+	"patients": {
+		"helicopter": "Prawdopodobieństwo skorzystania z HEMS",
+		"patient_allow_first_responder_chance": "Prawdopodobieństwo *First Responder*",
+		"patient_at_end_of_mission": "Pacjent pojawia się dopiero pod koniec misji!",
+		"patient_transport": "Prawdopodobieństwo transportu",
+		"possible_patient": "Maximum pacjentów",
+		"possible_patient_exact": "exact {n} pacjent | exact {n} pacjentów",
+		"possible_patient_min": "Minimum pacjentów",
+		"specializations": "Specializacja",
+		"title": "Pacjenci | {n} Pacjent | {n} Pacjentów"
+	},
+	"prerequisites": {
+		"fire_stations": "Fire Station | Fire Stations",
+		"max_police_stations": "Maximum Police Stations",
+		"max_rescue_stations": "Maximum Rescue Stations",
+		"police_helicopter_stations": "Police Helicopter Station | Police Helicopter Stations",
+		"police_stations": "Police Station | Police Stations",
+		"rescue_helicopter_stations": "Rescue Helicopter Count",
+		"rescue_stations": "Ambulance Station | Ambulance Stations",
+		"title": "Wymagania misji | Wymagania misji"
+	},
+	"prisoners": {
+		"max_possible_prisoners": "Maximum Prisoners",
+		"title": "Prisoner | {n} Prisoner | {n} Prisoner"
+	},
+	"vehicles": {
+		"captions": {
```

Comment:
I don't know if you want to translate this file directly or later?
You need to translate these captions here: 
![afbeelding](https://user-images.githubusercontent.com/65117490/100236015-676a8500-2f2d-11eb-888c-cebae66447d3.png)